src/Control/Inciter/Options/Problem.h

```diff
@@ -26,6 +26,8 @@ enum class ProblemType : uint8_t { USER_DEFINED=0,
                                    SHEAR_DIFF,
                                    DIR_NEU,
                                    VORTICAL_FLOW,
+                                   NL_ENERGY_GROWTH,
```

Comment:
This is indeed missing and will be documented next when I will also add regression tests. For now, if it does not break CI, will merge.